src/fields.rs

```diff
@@ -1163,6 +1163,18 @@ mod tests {
 		}
 		*/
 	}
+
+	#[test]
+	fn matched_slice() {
+		// This is a regression test to defend against a case where exact-size slices were produced
```

Comment:
I suppose that "exact-size slices" is rather vague wording. Would this comment be better as the following ?
```
// This is a regression test to defend against a case where slices fitting exactly
// into one `BitStore` element were represented by `Domain::Region` instead
// of `Domain::Enclave`.
```